docs/Reference/API-Methods.md

```diff
@@ -5145,12 +5145,6 @@ Distributes a signed, RLP encoded
 Returns the private transaction count for the specified account and
 [group of sender and recipients].
 
-!!! important
-
-    If sending more than one transaction to be mined in the same block (that is, you are not
-    waiting for the transaction receipt), you must calculate the private transaction nonce outside
-    Besu.
-
```

Comment:
I removed this because I was thinking it's not relevant to priv_getEeaTransactionCount (nonce isn't an input to this function) but is it meant to mean "use this method to calculate the nonce if you need to"?

Reply:
It actually means the opposite - if you're sending more than one transaction per block, this function won't return the correct nonce for you (ie, don't use it).   We could update it to: 

If sending more than one transaction to be mined in the same block (that is, you are not
waiting for the transaction receipt), you must calculate the private transaction nonce outside
Besu instead of using `priv_getEeaTransactionCount`.

Reply:
done